onnxruntime/python/tools/tensorrt/perf/perf_utils.py

```diff
@@ -8,6 +8,12 @@
 debug = False
 debug_verbose = False 
 
+def find(regex_string): 
```

Comment:
allows for change 
-        p1 = subprocess.Popen(["find", ".", "-name", "output*"], stdout=subprocess.PIPE)
-        p2 = subprocess.Popen(["sort"], stdin=p1.stdout, stdout=subprocess.PIPE)
-        stdout, sterr = p2.communicate()
-        stdout = stdout.decode("ascii").strip()
-        output_data = stdout.split("\n")
to
+        output_data = find("output*")